packages/docusaurus-theme-classic/src/theme/DocPage/index.tsx

```diff
@@ -52,86 +53,93 @@ function DocPageContent({
   }, [hiddenSidebar]);
 
   return (
-    <Layout
-      key={isClient}
-      searchMetadatas={{
-        version,
-        tag: docVersionSearchTag(pluginId, version),
-      }}>
-      <div className={styles.docPage}>
-        {sidebar && (
-          <div
-            className={clsx(styles.docSidebarContainer, {
-              [styles.docSidebarContainerHidden]: hiddenSidebarContainer,
-            })}
-            onTransitionEnd={(e) => {
-              if (
-                !e.currentTarget.classList.contains(styles.docSidebarContainer)
-              ) {
-                return;
-              }
+    <>
+      <Head>
+        <style>{`.main-wrapper { display: flex; }`}</style>
```

Comment:
Since we only have one CSS bundle for entire site, all the styles defined in the components become globally available. And therefore, I had to add this rule directly in the component via the `style` tag (because it should only be applied to docs). I don't like this approach, but are there any better options?

Reply:
Why not add the custom `className` to the Docs container/wrapper and just style via this selector?

Reply:
That's right, I completely forgot about this wonderful prop, thanks for the tip!